docs/performance.md

```diff
@@ -0,0 +1,101 @@
+# Performance
+
+By default, FrankenPHP tries to offer a good compromise between performance and ease of use.
+However, it is possible to slightly improve performance using the appropriate configuration.
+
+## Number of Threads and Workers
+
+By default, FrankenPHP starts 2 times more threads and workers (in worker mode) than the available numbers of CPU.
+
+The appropriate values depend heavily on how your application is written, what it does and your hardware.
+We strongly recommend changing these values.
+
+To find the right values, it's best to try out different values and run load tests simulating real traffic.
+[k6](https://k6.io) and [Gatling](https://gatling.io) are good tools for this.
+
+To configure the number of threads, use the `num_threads` option of the `php_server` and `php` directives.
+To change the number of workers, use the `num` option of the `worker` section of the `frankenphp` directive.
+
+## Worker Mode
+
+Enabling [the worker mode](worker.md) will dramatically improve the performance,
+but your app must be adapted to be compatible with this mode:
+you need to create a worker script and to be sure that the app is not leaking memory.
+
+## Prefer Not Using musl Builds
+
+The static binaries we provide and the Alpine Linux variant of the official Docker images
+are using [the musl libc](https://musl.libc.org/).
+
+PHP is known to be significantly slower when using this alternative C library instead of the traditional GNU library,
+especially when compiled in ZTS mode (thread-safe), which is required for FrankenPHP.
+
+Some bugs also only happen when using glibc.
+
+In production environement, we strongly recommend to use the glibc.
```

Comment:
Don't you mean `musl` here?

```suggestion
Some bugs also only happen when using musl.

In production environement, we strongly recommend to use the musl.
```

Reply:
bugs happen only with musl and we recommend glibc in prod, sorry :D